Userland/Utilities/wc.cpp

```diff
@@ -58,9 +58,21 @@ static ErrorOr<Count> get_count(StringView file_specifier)
     }
 
     auto file = TRY(Core::InputBufferedFile::create(maybe_file.release_value()));
-
     count.name = file_specifier;
 
+    if (only_bytes) {
+        auto seek_or_error = file->seek(0, SeekMode::FromEndPosition);
+        if (!seek_or_error.is_error()) {
+            count.bytes = seek_or_error.release_value();
+            return count;
+        }
+
+        // NOTE: If the error is not ESPIPE, then return an error, or otherwise
+        // just handle the input as a non-seekable stream.
```

Comment:
As usual, not really un-obvious enough to warrant specifically pointing the reader at the comment.

```suggestion
        // Handle the file as non-seekable if the error indicates it as such.
```